Misc/NEWS.d/next/Library/2022-11-27-07-00-13.gh-issue-99811.Vi7qsJ.rst

```diff
@@ -0,0 +1 @@
+Use correct variable in logging.StringTemplateStyle when searching for asctime.
```

Comment:
Actually, as this is an internal implementation detail, I don't think a NEWS entry is merited. I've added the "skip-news" label to this PR so it shouldn't fail for missing NEWS. Please remove that file. Otherwise fine - thanks for spotting it!

Reply:
Will do.  It's my first contribution to Python so I figured I'd follow instructions to the letter, then go from there :)